server/service/integration_mdm_profiles_test.go

```diff
@@ -2346,12 +2346,15 @@ func (s *integrationMDMTestSuite) TestBatchSetMDMAppleProfiles() {
 
 	// payloads with reserved types
 	for p := range mobileconfig.FleetPayloadTypes() {
+		if p == mobileconfig.FleetCustomSettingsPayloadType {
+			continue
```

Comment:
Nit: It might be helpful to add a comment as to why this type is being skipped, perhaps with a reference to `file_vault_options_test` where test coverage for this payload type resides.